tesseract-server/src/handlers/logic_layer/aggregate.rs

```diff
@@ -206,3 +332,682 @@ pub fn logic_layer_aggregation(
         })
         .responder()
 }
+
+
+/// Generates a series of Tesseract queries from a single LogicLayerQueryOpt.
+/// This function contains the bulk of the logic layer logic.
+pub fn generate_ts_queries(
```

Comment:
Since this diff is so big, just want to check here if you made big changes to this fn. I feel like I remember some of this code from before.

Reply:
Not to this part of the function, just some small cleanup.